### ISSUE TITLE
Make units of force constant configurable

### ISSUE THREAD

**Issue by tgmaxson** [dnlbauer/WHAM#16]
What are the units of the force constant?  From reading the src I cannot tell if this should be in kcal/mol or kT units etc.

It would be nice if we could supply the value of kB on the CLI such that we can use whatever unit we would like (or have a table to lookup from such as https://en.wikipedia.org/wiki/Boltzmann_constant#Value_in_different_units)

**Comment by tgmaxson** [comment]
Nevermind, I found the unit.  Can we make this configurable?  I am not familiar with Rust in general but I think this is an easy change.

static k_B: f64 = 0.008_314_462_1; // kJ/mol*K

**Comment by dnlbauer** [comment]
You're correct that the force constant is currently set in kJ/mol*K. This was what I typically worked with when writing the tool.

One thing to consider is that if we change the constant to be in any unit, the meaning of temperature also changes from "Temperature is always in Kelvin" to "in the format of k_B". But this is just a semantic issue.

If someone is interested in submitting a pull request to implement this feature, it should be a relatively simple change.  kJ/mol*K should stay the default value to be backward compatible, though.
